src/AppBundle/Entity/TaskCollectionItem.php

```diff
@@ -11,8 +11,9 @@
  *
  * @ORM\Entity
  * @ORM\Table(name="task_collection_item", uniqueConstraints={
- *   @ORM\UniqueConstraint(name="task_collection_item_unique", columns={"parent_id", "task_id"})}
- * )
+ *   @ORM\UniqueConstraint(name="task_collection_item_unique", columns={"parent_id", "task_id"}),
+ *   @ORM\UniqueConstraint(name="task_collection_position_unique", columns={"parent_id", "position"})
```

Comment:
@alexsegura agree with this?

I think there are bugs that can cause items to have duplicate position in the same list (for instance two tasks at position 2)
-> need investigation and tests
```sql
select * from task_collection_item where task_id between 443 AND 450 order by position;
 id  | parent_id | task_id | position
-----+-----------+---------+----------
 278 |        24 |     446 |        0
 280 |        24 |     448 |        2
 276 |        24 |     444 |        4
 281 |        24 |     449 |        4
 279 |        24 |     443 |        4
 282 |        24 |     450 |        5
 301 |        24 |     447 |       24
```